Admin tool decluttering

MrKicker opened the pull request:
Various tweaks to help declutter the admin-tab and the admin context menu.

:cl:
admin: Added Paralyze option to player panel.
admin: Removed Paralyze from admin tab (still in right-click and player panel).
admin: Moved 'View Persistant Data' to Secrets->Investigations.
admin: Removed 'Show Player Panel' from admin tab (right-click still works, 'player panel' unchanged).
admin: Removed skills from admin tab (still in player panel / right-click menu).
/:cl:

AlexNayl committed:
various admin tool tweaks

Mordeth221 commented:
> admin: Removed skills from admin tab and right click (still in player panel).

I don't like the removal from right click.

MrKicker commented:
> 
> 
> > admin: Removed skills from admin tab and right click (still in player panel).
> 
> I don't like the removal from right click.

Reverted that part, thanks

Spookerton commented:
Travis is yours - bad indentation in `admin_verbs.dm`

Probably the line with `//datum/admins/proc/view_persistent_data//Removed to de-clutter admin tab` - remove it instead.

Spookerton commented on code/modules/admin/admin_verbs.dm:
```diff
@@ -291,15 +291,15 @@ var/list/admin_verbs_mod = list(
 	/client/proc/check_fax_history,
-	/datum/admins/proc/paralyze_mob,
-	/datum/admins/proc/view_persistent_data
+	/datum/admins/proc/paralyze_mob,	// right-click paralyze
+	//datum/admins/proc/view_persistent_data//Removed to de-clutter admin tab
 )
```
```suggestion
	/datum/admins/proc/paralyze_mob // right-click paralyze
)
```

Spookerton replied:
probably this.

SierraKomodo commented on code/modules/admin/admin_verbs.dm:
```diff
@@ -1,6 +1,6 @@
 //admin verb groups - They can overlap if you so wish. Only one of each verb will exist in the verbs list regardless
 var/list/admin_verbs_default = list(
-	/datum/admins/proc/show_player_panel,	//shows an interface for individual players, with various links (links require additional flags,
+	/datum/admins/proc/show_player_panel, //shows an interface for individual players, with various links (links require additional flags), right-click player panel
```
```suggestion
	/datum/admins/proc/show_player_panel, //shows an interface for individual players, with various links (links require additional flags), right-click player panel,
```

SierraKomodo commented on code/modules/admin/admin_verbs.dm:
```diff
@@ -75,7 +75,7 @@ var/list/admin_verbs_admin = list(
 	/client/proc/toggleghostwriters,
 	/client/proc/toggledrones,
-	/datum/admins/proc/show_skills,
+	/datum/admins/proc/show_skills, //Right click skill menu
```
```suggestion
	/datum/admins/proc/show_skills, //Right click skill menu,
```

SierraKomodo commented on code/modules/admin/admin_verbs.dm:
```diff
@@ -291,15 +291,14 @@ var/list/admin_verbs_mod = list(
 	/client/proc/dsay,
-	/datum/admins/proc/show_skills,
-	/datum/admins/proc/show_player_panel,
+	/datum/admins/proc/show_skills,	// Right-click skill menu
```
```suggestion
	/datum/admins/proc/show_skills,	// Right-click skill menu,
```

SierraKomodo commented on code/modules/admin/admin_verbs.dm:
```diff
@@ -291,15 +291,14 @@ var/list/admin_verbs_mod = list(
-	/datum/admins/proc/show_skills,
-	/datum/admins/proc/show_player_panel,
+	/datum/admins/proc/show_skills,	// Right-click skill menu
+	/datum/admins/proc/show_player_panel,// right-click player panel
```
```suggestion
	/datum/admins/proc/show_player_panel,// right-click player panel,
```

SierraKomodo commented on code/modules/admin/admin_verbs.dm:
```diff
@@ -291,15 +291,14 @@ var/list/admin_verbs_mod = list(
 	/client/proc/check_fax_history,
-	/datum/admins/proc/paralyze_mob,
-	/datum/admins/proc/view_persistent_data
+	/datum/admins/proc/paralyze_mob // right-click paralyze
```
```suggestion
	/datum/admins/proc/paralyze_mob // right-click paralyze,
```

SierraKomodo reviewed (commented):
This might fix the travis fails - Noticed that every other comment in these lists end in a comma

MrKicker commented:
Fixed thanks to the help of Sierra, will squash
